public/modules/custom/grants_handler/src/ApplicationHandler.php

```diff
@@ -2063,4 +2091,29 @@ public static function isProduction(string $appEnv): bool {
     return in_array($appEnv, $proenvs);
   }
 
+  /**
+   * Tries to find latest webform for given application ID.
+   *
+   * @param mixed $id
+   *   Application id (eg. KASKOIPLISA)
+   *
+   * @return \Drupal\webform\Entity\Webform|null
+   *   Return webform object if found, else null.
+   */
+  public static function getLatestApplicationForm($id): Webform|NULL {
+    $webforms = \Drupal::entityTypeManager()
+      ->getStorage('webform')
+      ->loadByProperties([
+        'third_party_settings.grants_metadata.applicationType' => $id,
+        'archive' => FALSE,
```

Comment:
Muualla stringi tai muuttujan nimi on ollut 'archived'. Onko tämä tarkoituksella näin?

Reply:
archived on meidän 3rd party settings, tässä katsotaan webformin omaa sisäänrakennettua.